app/src/main/java/org/wikipedia/edit/insertmedia/InsertMediaActivity.kt

```diff
@@ -152,10 +153,23 @@ class InsertMediaActivity : BaseActivity() {
                 true
             }
             R.id.menu_save -> {
+                if (viewModel.invokeSource == Constants.InvokeSource.EDIT_ADD_IMAGE && viewModel.selectedImage != null) {
```

Comment:
Can we assign a value for `viewModel.selectedImage` before the if statement so that we do not have to put `?` or `!!` for the parameters?
`val selectedImage = viewModel.selectedImage`